.docker/ci/Dockerfile

```diff
@@ -29,22 +31,56 @@ RUN \
         wget \
         # Required for rosdep command
         sudo \
-        # Preferred build tools
-        python-catkin-tools \
         clang clang-format-3.9 clang-tidy clang-tools \
-        ccache && \
-    #
-    # Download all dependencies of MoveIt!
-    rosdep update && \
-    rosdep install -y --from-paths . --ignore-src --rosdistro ${ROS_DISTRO} --as-root=apt:false && \
+        ccache \
+        curl gnupg2 lsb-release \
+        # Some dependencies for working with ROS2
+        build-essential \
+        cmake \
+        git \
+        python3-colcon-common-extensions \
+        python3-pip \
+        python-rosdep \
+        python3-vcstool && \
     #
-    # Remove the source code from this container
-    # TODO: in the future we may want to keep this here for further optimization of later containers
-    cd .. && \
-    rm -rf src/ && \
+    # Upgrade Pip
+    python3 -m pip install --upgrade pip && \
     #
-    # Clear apt-cache to reduce image size
-    rm -rf /var/lib/apt/lists/*
+    # install some pip packages needed for testing
+    python3 -m pip install -U \
+      argcomplete \
+      flake8 \
+      flake8-blind-except \
+      flake8-builtins \
+      flake8-class-newline \
+      flake8-comprehensions \
+      flake8-deprecated \
+      flake8-docstrings \
+      flake8-import-order \
+      flake8-quotes \
+      git+https://github.com/lark-parser/lark.git@0.7d \
```

Comment:
is this a TODO?

Reply:
This is from the ROS2 install from source instructions.

Reply:
I have to go through and prune these